testing/rabbitmq/rabbitmq.initd

```diff
@@ -0,0 +1,38 @@
+#!/sbin/openrc-run
+
+pidfile=/var/run/rabbitmq/rabbitmq.pid
+name=rabbitmq
+daemon=/usr/sbin/rabbitmq-server
+
+depend() {
+	need net localmount
+	after firewall
+}
+
+start() {
+	ebegin "Starting ${name}"
+	# We need to test if /var/run/rabbitmq exists, since scirpt will fail if it does not
+	[ ! -e /var/run/rabbitmq ] && \
+		mkdir /var/run/rabbitmq && \
+		chown rabbitmq:rabbitmq /var/run/rabbitmq
+
+    
+    
+	start-stop-daemon --start \
+		--env RABBITMQ_PID_FILE=$pidfile \
+		--pidfile $pidfile \
+		--exec /bin/sh -- -c "$daemon \
+		> /var/log/rabbitmq/startup_log \
+		2> /var/log/rabbitmq/startup_err 0<&- &"
```

Comment:
I’m quite sure (but not 100%) that output redirection doesn’t work here. Moreover, `start-stop-daemon` provides options for them: `--stdout` and `--stderr`.
Using `&` here to put the process into background is absolutely wrong, it defeats the purpose of start-stop-daemon.